MAP: Портируем оффовский Colossus и меняем его. Добавляем профессию пилота.

Erring opened the pull request:
## Описание / Что этот PR делает
Колосс - это универсальный боевой корабль. Основной выездной отряд интеков, который может сразу идти в бой. Удобен для контрактов любых, удобен для аутизма, удобен для жизни, удобен по своей природе универсала. 
### Данный пр подтягивает изменения оффов (его реворк) и чутка меняет внешность, внутрянку. А также добавляет новую роль пилота для Интеков для этого корабля.
Теперь колосс по сравнению с целадоновским в разы больше, имеет суб, более красивый и удобный!
И ЕСТЬ БАНЯ. И в ней сюрприз.
### Эквип:
По деньгам 3 тысячи на столе + 2 тысячи в консоли. Имеют много денег раундстартом ради гибкости в эквипе.
СКМ, ф4, 2 монгреля. 3 магазина каждый. ф90 с 2 магазинами. Для кэпа пистолет на .44 в кейсе. 3 боевых ножа и 2 томагавка от гаммы с другим названием и описанием. 
4 фонарика.
По патронам: 1 коробка для каждой пушки кроме как монгрелей, которые имеют целый кейс патронов.
### Броня:
2 медиум с x11, 2 поинтмена пока что, 4 сек рига. 3 обычных комплекта брони.
Аптечек нет, есть хирургический набор, хирург. стол без компьютера.
### По ролям:
авангард, МАА, пилот суба (офицеры)
3 энфорсера, 2 ассистента, 1 полемед, 2 инженера. 
## Причина создания ПР / Почему это хорошо для игры
Колосс был стар, его ни разу особо не меняли, его никогда не брали, плюс он был даже слабым.
Данный пр подтягивает его до стандартов оффов в плане внутрянки и подтягивает его до стандартов целадона (если те есть). Что есть хорошо. 
## Демонстрация изменений / Тестирование
<details><summary>Старый скрин Ингейм</summary>
<img width="959" height="681" alt="image" src="https://github.com/user-attachments/assets/54027925-7861-4f31-8edc-f5c10167af6d" />
</details>
<details><summary>Скрин SDMM</summary>
<img width="1408" height="928" alt="image" src="https://github.com/user-attachments/assets/c77e2694-0b04-46cd-aeee-97a11d3bd1c6" />
</details>
<details><summary>Скрин SDMM с зонами</summary>
<img width="1408" height="928" alt="image" src="https://github.com/user-attachments/assets/c1eea124-ab4d-4af6-952c-32a3646d90ba" />
</details>

## Список изменений

```yml
🆑
add: Роль пилота Интеков
map: Добавлен суб. Перенесен с оффов и переделан Колосс Интеков.
/🆑
```

Erring committed:
Колосса раздуло

Erring committed:
фиксим зоны и немного платформы

Erring committed:
Правим камеры и что-то ещё

KOCMODECAHTHUK reviewed (dismissed):
по коду все ок

Erring committed:
Merge remote-tracking branch 'upstream/beta-dev' into off-colossus-rework

Erring committed:
fixing zones

Erring committed:
Merge remote-tracking branch 'upstream/beta-dev' into off-colossus-rework

Erring committed:
Окна, стул в кэпской + фикс скма

Erring committed:
remaking sauna

Cuildipie commented:
Ангар пиздатый. Наконец-то не 1 тайл прохода. Та и в целом шип прикольный, теперь оправдывает своё название, лайк. Надеюсь проверил все три док порта на работоспособность и габариты.

Erring committed:
кредиты: 5к-> 3к + докинг порт

Меняем положение докинг порта для возможности садиться к докинг портам кораблей (если те могут его по размеру содержать).
Пиксель шифтим интерком.
Меняем с 5к до 3к
